components/Autocomplete/index.jsx

```diff
@@ -0,0 +1,260 @@
+import React, {PropTypes} from 'react';
+import Autosuggest from 'react-autosuggest';
+import Scroll from 'react-scroll';
+
+import styles from './index.scss';
+
+const isMobile = window.outerWidth < 550;
+
+export default class Autocomplete extends React.Component {
+  static propTypes = {
+    data: PropTypes.object,
+    disabled: PropTypes.bool,
+    error: PropTypes.string,
+    id: PropTypes.string.isRequired,
+    inputStyle: PropTypes.string,
+    label: PropTypes.string.isRequired,
+    labelHidden: PropTypes.bool,
+    name: PropTypes.string.isRequired,
+    noSuggestionText: PropTypes.string.isRequired,
+    onGetSuggestions: PropTypes.func,
+    placeholder: PropTypes.string,
+    query: PropTypes.object,
+    required: PropTypes.bool,
+    requiredLabel: PropTypes.string,
+    scrollTo: PropTypes.bool,
+    submitOnEnter: PropTypes.bool,
+    suggestionsFooter: PropTypes.any,
+    value: PropTypes.string
+  };
+
+  static defaultProps = {
+    error: null,
+    inputStyle: 'inline',
+    noSuggestionText: 'No results found',
+    placeholder: '',
+    query: {},
+    scrollTo: true
```

Comment:
`scrollTo` is nice but I would set it to `false` by default for now.
This is so that the default options guarantee a unified user experience - the scroll needs to be implemented for every component that has a dropdown and it would be weird if it scrolled down or up exclusively for `Autocomplete`.